src/client/java/teammates/client/scripts/sql/SeedDb.java

```diff
@@ -372,14 +378,14 @@ private void seedNotificationAccountAndAccountRequest(int constReadNotificationS
                 log(e.toString());
             }
         }
+    }
 
+    private void seedAccountRequests() {
         for (int i = 0; i < MAX_ENTITY_SIZE; i++) {
```

Comment:
Will seeding `MAX_ENTITY_SIZE` of account requests impact performance?

Reply:
Nope its relatively quick, I'm using it for a non-course script which we will he running tonight, so we can comment it after that if you think its slow